### DISCUSSION TITLE
AD Combine Unload Precall

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2612] [Q&A]
IS the AD precall command triggering CP to immediately put the pipe out? or is this AD overriding CP? I would love to be able to turn it off until the trailer is near or CP has made the pocket. On large fields the combine always gets stuck on trees doing the first run as soon as the precall level is hit as the pipe is put out. If it's AD, has anyone here found the code which controls it?

**Comment by Tensuko** [comment]
Cp doesn't put the pipe out, thats an overwrite by AD.
